### DISCUSSION TITLE
Upload and archive with cli

### DISCUSSION THREAD

**Discussion by enboig** [immich-app/immich#7678] [Feature Request]
### The feature

I am adding lot's of old photos from my collection. I would like to add them to an album and become automatically archived. Is there a way to achieve this?

Also adding to more than one album would be welcome.

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by ljlzzzzz** [comment]
https://immich.app/docs/features/command-line-interface, you can try this, i hope it will help you
